dlp/jobs_test.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import jobs
+
+import pytest
+import os
+
+GCLOUD_PROJECT = os.getenv('GCLOUD_PROJECT')
```

Comment:
I believe we prefer GOOGLE_CLOUD_PROJECT for both sides

Reply:
I agree in principle, but unfortunately the other languages such as Node are still using GCLOUD_PROJECT; perhaps we should standardize across all languages at once rather than try to change it in this PR.

Reply:
@jonparrott PTAL
@broady PTAL

We were supposed to standardized on GOOGLE_CLOUD_PROJECT over two years ago.